Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Use the official Ruby 3.2 image as a base image
+ARG RUBY_VERSION=3.2
+FROM ruby:${RUBY_VERSION}-bullseye
+
+# Set the working directory
+WORKDIR /usr/src/app
+
+# Install system packages
+RUN apt-get update -qq && \
+    apt-get install -y default-mysql-client postgresql postgresql-contrib vim && \
```

Comment:
I'm installing `postgresql` too. With the hope I'll have a future PR where I can add it via `docker-compose.yml`. It doesn't hurt anything currently to leave it in. I'm also installing `vim` because someone may want to edit files inside the container.